libethereum/ExtVM.cpp

```diff
@@ -33,7 +33,7 @@ namespace // anonymous
 static unsigned const c_depthLimit = 1024;
 
 /// Upper bound of stack space needed by single CALL/CREATE execution. Set experimentally.
-static size_t const c_singleExecutionStackSize = 6 * 1024;
+static size_t const c_singleExecutionStackSize = 13 * 1024;
```

Comment:
Please could you add some comment here about where "13" has come from.    No doubt a lot of experimental effort into that change, but all of that context is lost with this commit, so anybody in the future has no idea where "13" came from, or what led to that.

Just some comments here would be very helpful.